### ISSUE TITLE
Do we need to have a real fsi.exe if we want to evaluate a script on the fly

### ISSUE THREAD

**Issue by forki** [fsharp/FSharp.Compiler.Service#877]
https://fsharp.github.io/FSharp.Compiler.Service/interactive.html#Starting-the-F-interactive

![image](https://user-images.githubusercontent.com/57396/46672878-dca55f80-cbd8-11e8-85e9-d3f7b88122e2.png)

In this sample we need the path of a fsi.exe in order to evaluate code with the FCS. That's a bit surprising since the idea of the FCS is to use it on the fly as a lib, right?

**Comment by matthid** [comment]
Afaik that parameter is a dummy value and can be set to anything (it might change how errors are reported if used at all)

**Comment by dsyme** [comment]
Yeah it's a dummy value. It's just one of those whacky things that never seemed to be worth changing.

**Comment by goswinr** [comment]
maybe udating the documnetation is good enough
also mentioned here:
https://github.com/fsharp/FSharp.Compiler.Service/issues/420
